cvl/Makefile

```diff
@@ -18,76 +18,93 @@
 ################################################################################
 
 GO?=go
+TOPDIR ?= ..
+BUILD_DIR:=$(TOPDIR)/build/cvl
+FORMAT_CHECK = $(BUILD_DIR)/.formatcheck
 SRC_FILES=$(shell find . -name '*.go' | grep -v '_test.go' | grep -v '/tests/')
 TEST_FILES=$(wildcard *_test.go)
-TOP_DIR := ..
-BUILD_DIR:=$(TOP_DIR)/build/cvl
-FORMAT_CHECK = $(BUILD_DIR)/.formatcheck
+CVL_TEST_DIR = $(TOPDIR)/build/tests/cvl
+CVL_TEST_BIN = $(CVL_TEST_DIR)/cvl.test
+CVL_TEST_SCHEMA_DIR = $(CVL_TEST_DIR)/testdata/schema
+CVL_TEST_CONFIG = $(CVL_TEST_DIR)/cvl_cfg.json
+CVL_TEST_DB_CONFIG = $(CVL_TEST_DIR)/database_config.json
```

Comment:
Is this test db intended to be the one generated by db.config.py?
```
        # Update unixsocket path in database_config.json
        tools/test/dbconfig.py -o build/tests/database_config.json
        export DB_CONFIG_PATH=${PWD}/build/tests/database_config.json
```

Reply:
yes